apps/mull-ui/src/app/pages/login/login.scss

```diff
@@ -4,97 +4,101 @@
   display: grid;
   grid-template-columns: 1fr;
```

Comment:
I haven't used grid too much but this seems unecessary. Double check if remove if it is
```suggestion
```